packages/web-twig/src/Resources/components/UNSTABLE_EmptyState/stories/EmptyStateDefault.twig

```diff
@@ -1,48 +1,54 @@
+{% set slotContent %}
+    <div style="border: 1px solid #E9E9E9; border-radius: 10px; padding: 12px ">
+        <Icon name="search" />
+    </div>
+{% endset %}
+
 <UNSTABLE_EmptyState
```

Comment:
To be honest, I think this example should not be possible. E. g. use children and build it by own or use the props.

Reply:
refactored (different solution)